docs/contributors.json

```diff
@@ -18,4 +18,4 @@
   "ydcjeff",
   "11joselu",
   "wheatjs"
-]
\ No newline at end of file
+]
```

Comment:
```suggestion
]

```

Reply:
@veritem could you remove this file from the change set? I don't know why I'm getting an error when trying to check out the branch